scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -26,9 +26,23 @@ namespace <%= namespace %> {
             }
         }
 
-        private CartLineItems _LineItems;
+        private bool IsSaved {
+            get {
+                return IsCreated && LineItems.IsSaved;
+            }
+        }
 
+        private bool IsCreated {
+            get {
+                return CurrentCheckout != null;
+            }
+        }
+
+        private const float POLL_DELAY_SECONDS = 0.5f;
+        private CartLineItems _LineItems;
+        private List<String> DeletedLineItems = new List<string>();
         private ShopifyClient Client;
+        private Checkout CurrentCheckout = null;
```

Comment:
👍  Jake do you know any good linting tools that are not tied to a specific C# dev environment? I think you're right it would be nice to do a linting pass at some point.

I've been doing `_LineItems` for cases where there's an accessors that mutates or exposes the variable. I've used `PascalCase` for class variables since that seems to be the way to go with C# projects.